WooCommerce/Classes/ViewModels/ShippingProvidersViewModel.swift

```diff
@@ -7,12 +7,13 @@ import Yosemite
 final class ShippingProvidersViewModel {
     let order: Order
 
+    /// Title of view displaying all available Shipment Tracking Providers
     let title = NSLocalizedString("Shipping Providers",
                                   comment: "Title of view displaying all available Shipment Tracking Providers")
 
     /// ResultsController: Surrounds us. Binds the galaxy together. And also, keeps the UITableView <> (Stored) StorageShipmentTrackingProviderGroup in sync.
     ///
-    private(set) lazy var resultsController: ResultsController<StorageShipmentTrackingProvider> = {
```

Comment:
Was the private setter removed intentionally?

Reply:
Yes, to make the property fully private.